project_forecast_line_deadline/models/project_task.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Therp BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    # Make forecast end date computable and stored
+    forecast_date_planned_end = fields.Date(
+        compute="_compute_forecast_date_planned_end", inverse=lambda x: None, store=True
+    )
+
+    @api.depends("date_deadline")
+    def _compute_forecast_date_planned_end(self):
+        """Set forecast end to be equal with date deadline"""
+        for this in self:
+            this.forecast_date_planned_end = this.date_deadline
```

Comment:
Another question maybe: let's say someone has 10000 tasks in the system and does already have `project_forecast_line` installed.

- Upon installation, is this field recomputed? (I guess not, because it already exists?)
- Do we want it to recompute upon installation, or not really (I'm guessing not, only when deadlines are changed? But I'm not sure. @gjotten )

Reply:
It is not recomputed upon installation, and we need to keep it that way, because user can either write manually on forecast_date_planned_end, or edit it indirectly by setting date_deadline.